Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -866,9 +866,14 @@ private List<string> GetDirectories(string folder)
 
         public void Exclude(bool delete)
         {
-            if (delete && !View.ShowDeleteDialog(SelectedItems))
+            // We don't want to show a delete confirmation for any items outside the project folder
+            var filteredItems = new List<IProjectItem>(SelectedItems.Where(i => !i.OriginalPath.Contains("..")));
+
+            if (filteredItems.Count > 0 && delete && !View.ShowDeleteDialog(filteredItems))
```

Comment:
We might want to just not include a delete option for ".." folders all together.